### PR TITLE
RF-GFN changes to evaluation

### PR DESCRIPTION
**Pull request by michalkoziarski**

### CODE REVIEW

File: config/experiments/tree.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  random_action_prob: 0.1
+  optimizer:
+    batch_size:
+      forward: 100
+      backward_dataset: 0
+      backward_replay: 0
+    lr: 0.00001
+    z_dim: 16
+    lr_z_mult: 1000
+    n_train_steps: 40000
```
Comment (alexhernandezgarcia) [inline config/experiments/tree.yaml:26]: 40000 steps by default may be too much?

Comment (michalkoziarski) [reply config/experiments/tree.yaml:26]: Yeah, I wasn't sure how quick convergence to expect / how appropriate lowering the number of steps would be without tuning the remaining parameters (these are taken from alanine dipeptide experiment), so I just started with a long training and figured I would lower it later. But feel free to propose any other value.

File: gflownet/envs/tree.py [hunk 2]
```diff
@@ -1070,3 +1130,37 @@ def plot(self) -> None:
             node_size=800,
         )
         plt.show()
+
+    @staticmethod
+    def _compute_scores(
```
Comment (alexhernandezgarcia) [inline gflownet/envs/tree.py]: Some docstring would be great.

**Review by alexhernandezgarcia — APPROVED** [review]
It looks good to me! It would be nice to add some missing docstring in some methods of the tree environment, but I am happy to merge asap regardless.

**Pull request merged**